Riot/Modules/Settings/Security/SecurityViewController.m

```diff
@@ -292,14 +292,19 @@ - (void)updateSections
     Section *pinCodeSection = [Section sectionWithTag:SECTION_PIN_CODE];
     
     // Header title
-    NSString *pinCodeSectionHeaderTitleFormat = NSLocalizedStringFromTable(@"pin_protection_settings_section_header_x", @"Vector", nil);
-    NSString *pinCodeSectionHeaderTitle = [NSString stringWithFormat:pinCodeSectionHeaderTitleFormat, [PinCodePreferences shared].localizedBiometricsName];
-    pinCodeSection.headerTitle = pinCodeSectionHeaderTitle;
+    if ([PinCodePreferences shared].isBiometricsAvailable)
+    {
+        NSString *pinCodeSectionHeaderTitleFormat = NSLocalizedStringFromTable(@"pin_protection_settings_section_header_x", @"Vector", nil);
+        NSString *pinCodeSectionHeaderTitle = [NSString stringWithFormat:pinCodeSectionHeaderTitleFormat, [PinCodePreferences shared].localizedBiometricsName];
+        pinCodeSection.headerTitle = pinCodeSectionHeaderTitle;
+    }
```

Comment:
Should there be a header in an else here? Just "PIN"?

Reply:
I'd probably keep one to be consistent with the rest of the screen. Not that I feel that strongly about it. :)

Reply:
Okay, will do 👍